Assets/Scripts/Model/Players/EasyAI.cs

```diff
@@ -8,21 +8,21 @@ public class EasyAI : AI
 	public override void Play(){
         InvokeRepeating("autoCoins", 0.0f, 1.0f);
         //create 20 units every wave
-	    Shop shop = GameObject.FindGameObjectWithTag("Shop").GetComponent<Shop>();
+	    UnitShop shop = GameObject.FindGameObjectWithTag("Shop").GetComponent<UnitShop>();
         if (unitsWave < 20)
```

Comment:
Seems all ok! Just to know, event if it's not a change on this PR, what is unitsWave? (I don't see where it comes from)